kubernetes/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterScheduler.scala

```diff
@@ -91,10 +108,10 @@ private[spark] class KubernetesClusterScheduler(conf: SparkConf)
       s"--class=${args.userClass}",
       s"--master=$kubernetesHost",
       s"--executor-memory=${driverDescription.mem}",
-      s"--conf=spark.executor.jar=$clientJarUri",
-      s"--conf=spark.executor.instances=$instances",
-      s"--conf=spark.kubernetes.namespace=$nameSpace",
-      s"--conf=spark.kubernetes.driver.image=$sparkDriverImage")
+      s"--conf spark.executor.jar=$clientJarUri")
+
+    submitArgs ++= conf.getAll.filter { case (name, _) => !confBlackList.contains(name) }
```

Comment:
Hi @erikerlandson and @foxish 

Thanks for writing this change! I was looking at this because it would fix an issue I was having.

But it seems that these two lines appending to submitArgs are missing in [the latest revision of KubernetesClusterScheduler.scala]( https://github.com/foxish/spark/blob/2c67db5760ffa1e0de4d9bd2edf422dd5d202f9f/resource-managers/kubernetes/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterScheduler.scala). I wonder if they were intentionally removed?

Thanks,
Kimoon